clang/lib/CIR/Dialect/Transforms/TargetLowering/RecordLayoutBuilder.cpp

```diff
@@ -33,7 +33,7 @@ class EmptySubobjectMap {
   uint64_t charWidth;
 
   /// The class whose empty entries we're keeping track of.
-  const StructType Class;
+  const StructType klass;
```

Comment:
I don't like the name `klass`.  I prefer `Class` or `classType`.  But my feelings aren't strong enough to request that the name be changed.

Reply:
+1 here!